Makefile

```diff
@@ -76,7 +76,7 @@ serve:
 	node server.js
 
 test:
-	./node_modules/.bin/jasmine-node test/katex-spec.js
+	./node_modules/.bin/jasmine-node test/*-spec.js
```

Comment:
That jasmine issue can indeed be reproduced, see https://github.com/mhevery/jasmine-node/issues/379#issuecomment-155170552. It seems many projects I've seen prefer mocha over jasmine, which would handle this situation more elegantly. And jasmine-node doesn't seem to have a lot of activity lately. In fact zero commits to the main branches in the past year. If you feel like switching, let me know.